getting-started-guide/event-data.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Event data"
+description: "This page explains the fundamentals of timestamped event data in Axiom."
+sidebarTitle: Event data
+---
+
+Axioms’s mission is to help you leverage the power of timestamped event data.
+
+Axiom considers timestamped event data to be the record of every digital interaction between human, sensor, and machine. Event data is the atomic unit of activity. For this reason, every function in any business with digital activity can benefit from leveraging event data.
+
+Each event is simply a structured record—composed of key-value pairs—that captures meaningful interactions or changes in state within a system. While these can appear in various forms, they usually contain the following:
+
+- **Timestamp**: When the event occurred.
+- **Attributes**: A set of key-value pairs offering details about the event context.
+- **Metadata**: Contextual labels and IDs that connect related events.
+
+<Note>
+The timestamped event data that Axiom works with is not to be confused with time-series event data.
```

Comment:
Without more context, I would omit this note. It doesn’t help the reader much imo. 

Usually, when using this kind of language, we’re creating a distinction between time series databases (like Timescale), and Axiom. 

I might draw on Neil’s idea that Axiom is suited to answering questions across operational (now, individual), analytical (recent history, population) and archival workloads, where time series databases might not have quite the same applied nature. But others at Axiom might be able to add more.